tests/test_command_repo_files.py

```diff
@@ -0,0 +1,149 @@
+import unittest
+from argparse import ArgumentParser
+from unittest.mock import Mock, patch
+
+from huggingface_hub.commands.repo_files import DeleteSubCommand, RepoFilesCommand
+
+from .testing_utils import DUMMY_MODEL_ID
+
+
+class TestRepoFilesCommand(unittest.TestCase):
+    def setUp(self) -> None:
+        """
+        Set up CLI as in `src/huggingface_hub/commands/huggingface_cli.py`.
+        """
+        self.parser = ArgumentParser("huggingface-cli", usage="huggingface-cli <command> [<args>]")
+        commands_parser = self.parser.add_subparsers()
+        RepoFilesCommand.register_subcommand(commands_parser)
+
+    @patch("huggingface_hub.commands.repo_files.HfApi.delete_files_r")
+    def test_delete(self, delete_files_r_mock: Mock) -> None:
```

Comment:
```suggestion
    @patch("huggingface_hub.commands.repo_files.HfApi.delete_files")
    def test_delete(self, delete_files_mock: Mock) -> None:
```

(+ update variables below)